lib/presentation/screens/main_screen/main_layout.dart

```diff
@@ -52,40 +44,16 @@ class _MainLayoutState extends State<MainLayout> {
             }
           },
           builder: (context, state) {
-            return Column(
+            return Stack(
+              alignment: Alignment.bottomCenter,
               children: [
-                spaces,
-                const AssistantWidget(),
-                spaceBetween,
-                RichText(
-                  text: TextSpan(
-                    children: <TextSpan>[
-                      TextSpan(text: LocaleKeys.you_already_got.tr(), style: CustomTheme().gotCupsTextProperty),
-                      TextSpan(text: ' ${state.counter}', style: CustomTheme().countTextProperty),
-                      TextSpan(text: '/$cupsMaxCount ', style: CustomTheme().partOfCountTextProperty),
-                      TextSpan(text: LocaleKeys.cups.tr(), style: CustomTheme().gotCupsTextProperty),
-                    ],
-                  ),
-                ),
-                betweenCounters,
-                Align(
-                  alignment: Alignment.topLeft,
-                  child: Text(
-                    '${state.counter * singleCupWeight}${LocaleKeys.ml.tr()}',
-                    style: CustomTheme().counterMl,
-                  ),
-                ),
-                spaceBetweenManAndMl,
-                Align(
-                  alignment: Alignment.topRight,
-                  child: PersonImageWidget.mainWay(),
-                ),
-                spaceBetween,
-                CustomButton(
-                  text: 'ad',
+                MainCounterButtonWidget(
```

Comment:
mby rename it as MainScreenButtonWidget? and dont forget to rename the file accordingly